### PR TITLE
LPS-85855 TrashIndexer tests

### PR DESCRIPTION
**Pull request by luanmaoski**

### CODE REVIEW

**Comment by luanmaoski** [comment]
ci:test:search

File: modules/apps/trash/trash-test/src/testIntegration/java/com/liferay/trash/search/test/TrashFixture.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+				RandomTestUtil.randomString(), ContentTypes.TEXT_PLAIN, null,
+				getServiceContext());
+
+			DLTrashLocalServiceUtil.moveFileEntryToTrash(
+				getUserId(), fileEntry.getRepositoryId(),
+				fileEntry.getFileEntryId());
+
+			return fileEntry;
+		}
+		catch (PortalException pe) {
```
Comment (ifnazar) [inline modules/apps/trash/trash-test/src/testIntegration/java/com/liferay/trash/search/test/TrashFixture.java]: you can remove this catch, you do not use

File: modules/apps/trash/trash-test/src/testIntegration/java/com/liferay/trash/search/test/TrashIndexerSearchTest.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+	public void setUp() throws Exception {
+		setUpUserSearchFixture();
+		setUpIndexedFieldsFixture();
+		setUpTrashEntryIndexerFixture();
+		setUpTrashEntryFixture();
+
+		_fileEntry = trashFixture.createFileEntry();
+		
+		_fileEntry = DLAppLocalServiceUtil.getFileEntry(
+				_fileEntry.getFileEntryId());
```
Comment (ifnazar) [inline modules/apps/trash/trash-test/src/testIntegration/java/com/liferay/trash/search/test/TrashIndexerSearchTest.java]: you do not need to get the object again

**Comment by luanmaoski** [comment]
ci:test:search

**Review by ifnazar — CHANGES_REQUESTED** [review]

File: modules/apps/trash/trash-test/src/testIntegration/java/com/liferay/trash/search/test/TrashIndexerSearchTest.java (hunk 2 above)
Comment (luanmaoski) [reply modules/apps/trash/trash-test/src/testIntegration/java/com/liferay/trash/search/test/TrashIndexerSearchTest.java]: I need. I am using the `path` field here: `Assert.assertEquals(searchTerm, document.get("path"));`, this field changes when the file is moved to the trash.

File: modules/apps/trash/trash-test/src/testIntegration/java/com/liferay/trash/search/test/TrashFixture.java (hunk 1 above)
Comment (luanmaoski) [reply modules/apps/trash/trash-test/src/testIntegration/java/com/liferay/trash/search/test/TrashFixture.java]: Alright.

**Comment by luanmaoski** [comment]
ci:test:search

**Comment by luanmaoski** [comment]
ci:test:search

**Comment by luanmaoski** [comment]
ci:test:search

**Comment by luanmaoski** [comment]
ci:test:search

**Comment by luanmaoski** [comment]
ci:test:search

**Comment by luanmaoski** [comment]
ci:test:search

**Pull request closed without merging**